CHANGELOG.md

```diff
@@ -3,6 +3,43 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.6.0] - 2024-05-06
+
+### Added
```

Comment:
Add observability support for langchain based examples

Reply:
Added it thanks.